mesa/time.py

```diff
@@ -23,23 +23,20 @@
 
 # Mypy; for the `|` operator purpose
 # Remove this __future__ import once the oldest supported Python is 3.10
```

Comment:
These two lines can also be removed

Reply:
The reveiw is showing a failed check at codecov/patch regarding lines i haven't touched or changed. Do you have any idea related to that?